src/Language/Haskell/TH/Lift/Generics.hs

```diff
@@ -174,15 +185,36 @@ genericLiftTypedCompatWithPkg = genericLiftTypedTExpWithPkg
 -- foo :: Foo
 -- foo = $(lift (Foo 1 'a' "baz"))
 -- @
+#if MIN_VERSION_base (4,9,0)
```

Comment:
One convention I like to use to make reading code with nested CPP easier is to indent nested CPP blocks a certain number of spaces. For example, since this `MIN_VERSION_base` is nested one level deep, I'd prefer writing it with one extra space:

```suggestion
# if MIN_VERSION_base (4,9,0)
```

Similarly for those nested two levels deep and so on.

Reply:
Will do.